packages/clerk-js/src/core/clerk.ts

```diff
@@ -1063,7 +1063,7 @@ export default class Clerk implements ClerkInterface {
           await this.#devBrowserHandler.setup();
         } else if (!isValidBrowserOnline()) {
           console.warn(err);
-          break;
+          return false;
```

Comment:
ℹ️ This is to return `false` when `loadInStandardBrowser` fails, so it reflects that the clerk-js has not loaded yet.